devon4ng-architecture/index.asciidoc

```diff
@@ -0,0 +1,70 @@
+= Understanding the architecture principles followed by devon4ng
```

Comment:
This is a long title. Maybe consider a shorter title and move the longer one to the subtitle

Reply:
@SchettlerKoehler Done.